src/Microsoft.Diagnostics.Monitoring.WebApi/DumpService.cs

```diff
@@ -82,7 +88,26 @@ public async Task<Stream> DumpAsync(IEndpointInfo endpointInfo, Models.DumpType
                 operationRegistration?.Dispose();
             }
 
-            return new AutoDeleteFileStream(dumpFilePath);
+            string resultPath = dumpFilePath;
+            string dac = null;
+            string dbi = null;
+            if (mode.HasFlag(Models.PackageMode.IncludeDacDbi))
+            {
+                try
+                {
+                    string runtimeDir = DacLocator.LocateRuntimeComponents(endpointInfo, out dac, out dbi);
```

Comment:
Looks like the return value is not needed. If it's not used anywhere else, please remove it from the method.